packages/api/src/platforms/vtex/resolvers/offer.ts

```diff
@@ -78,7 +79,18 @@ export const StoreOffer: Record<string, Resolver<Root>> = {
     }
 
     if (isOrderFormItem(root)) {
-      return root.sellingPrice / 1e2
+      return root.sellingPrice / 1e2 / root.unitMultiplier
+    }
+
+    return null
+  },
+  priceWithTaxes: (root) => {
+    if (isSearchItem(root)) {
+      return withTax(price(root), root.Tax, root.product.unitMultiplier)
+    }
+
+    if (isOrderFormItem(root)) {
+      return withTax(root.sellingPrice  / 1e2 / root.unitMultiplier, root.tax / 1e2, root.unitMultiplier)
```

Comment:
The `sellingPrice` is the only price that is returned already multiplied by the unitMultiplier. To ensure consistency, this value is being divided by the unitMultiplier to return the unit price, like the others prices.